### PR TITLE
comm file phpcs

### PR DESCRIPTION
**Pull request by ramico**
an intersting read
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841

### CODE REVIEW

File: includes/class-wc-siftscience-comm.php [hunk 1]
```diff
@@ -50,15 +51,15 @@ public function post_event( $data ) {
...
 				'$abuse_type' => 'payment_abuse',
 			);
 
 			$url  = str_replace( '{user}', rawurlencode( $user_id ), self::LABELS_URL );
 			$args = array(
 				'headers' => self::HEADERS,
 				'method'  => 'POST',
-				'body'    => $data
+				'body'    => $data,
 			);
```
Comment (ramico) [inline includes/class-wc-siftscience-comm.php:64]: testing GitHub website just ignore this comment

Comment (ramico) [reply includes/class-wc-siftscience-comm.php:64]: this is inconvenient

**Comment by ramico** [comment]
another test

File: includes/class-wc-siftscience-comm.php (hunk 1 above)
Comment (nabsul) [reply includes/class-wc-siftscience-comm.php:64]: I see this comment.

**Pull request merged**